pages/linux/gitk.md

```diff
@@ -0,0 +1,23 @@
+# gitk
+
+> A graphical git repository browser.
+
+- Show repository browser in current git repository:
```

Comment:
Perhaps `Show the repository browser for the current git repository` would read better?

Reply:
Yes. You're right.
`Syntax Error`